clang/test/FixIt/format.cpp

```diff
@@ -12,33 +12,21 @@ struct S {
   N::E Type;
 };
 
-using uint32_t = unsigned;
-enum class FixedE : uint32_t { Two };
-
 void a(N::E NEVal, S *SPtr, S &SRef) {
   printf("%d", N::E::One); // expected-warning{{format specifies type 'int' but the argument has type 'N::E'}}
```

Comment:
Were these `expected-warning` necessary on the Apple CI after the changes?

They seem to be failing in every configuration when we pick up this change in our CI.

I noticed that there was no "please test llvm" in this change, so the LLVM/Clang tests were not executed in this PR.